pkg/storaged/stratis-details.jsx

```diff
@@ -726,13 +909,21 @@ const StratisStoppedPoolSidebar = ({ client, uuid }) => {
 };
 
 export const StratisStoppedPoolDetails = ({ client, uuid }) => {
+    const key_desc = client.stratis_stopped_pool_key_description[uuid];
+    const clevis_info = client.stratis_stopped_pool_clevis_info[uuid];
+
+    const encrypted = key_desc || clevis_info;
+    const can_tang = encrypted && (!clevis_info || clevis_info[0] == "tang");
+    const tang_url = (can_tang && clevis_info) ? JSON.parse(clevis_info[1]).url : null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230808-072951-1e5d5bf6-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)